packages/core/package.json

```diff
@@ -26,7 +26,7 @@
   "devDependencies": {
     "@ava/typescript": "^1.1.1",
     "@nomiclabs/hardhat-etherscan": "^2.1.0",
-    "@openzeppelin/contracts": "^3.1.0",
+    "@openzeppelin/contracts": "^4.0.0",
```

Comment:
This looks unrelated to this PR, along with all of the changes to the contracts in `core`.